forest/willow/log_stats.py

```diff
@@ -437,121 +437,104 @@ def log_stats_main(
             "HOURLY_AND_DAILY, DAILY, HOURLY"
         )
 
-    os.makedirs(output_folder, exist_ok=True)
-
     if frequency == Frequency.HOURLY_AND_DAILY:
-        os.makedirs(output_folder + "/hourly", exist_ok=True)
-        os.makedirs(output_folder + "/daily", exist_ok=True)
+        frequencies = [Frequency.HOURLY, Frequency.DAILY]
+    else:
+        frequencies = [frequency]
+
+    os.makedirs(output_folder, exist_ok=True)
+    for freq in frequencies:
+        os.makedirs(f"{output_folder}/{freq.name.lower()}", exist_ok=True)
 
     # beiwe_id should be a list of str
-    if beiwe_id is None:
-        beiwe_id = [
-            i for i in os.listdir(study_folder)
-            if os.path.isdir(f"{study_folder}/{i}")
+    if beiwe_ids is None:
+        beiwe_ids = [
+            participant_id for participant_id in os.listdir(study_folder)
+            if os.path.isdir(f"{study_folder}/{participant_id}")
         ]
 
-    if len(beiwe_id) > 0:
-        for bid in beiwe_id:
-            logger.info("User: %s", bid)
+    # process the data for each participant in each frequency into a folder of
+    # the corresponding frequency.
+    for beiwe_id in beiwe_ids:
+        for freq in frequencies:
+            logger.info(f"({freq.name.lower()}) Participant: {beiwe_id}")
```

Comment:
We use `logger.info("%s", var)` style in log strings (https://forest.beiwe.org/en/latest/development.html#python)